Solutions/NonameSecurity/Data Connectors/Connector_RESTAPI_NonameSecurity.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "f66a9bee-323d-4280-8c3b-e95bc66db83a",
```

Comment:
Hey @nniryanb, there was no need to change this, this would remain as "ProviderNameApplianceName", so you can revert back this id.